migrations/versions/201510281025_38641fb3afac_add_participation_regforms.py

```diff
@@ -0,0 +1,25 @@
+"""Add participation regforms
+
+Revision ID: 38641fb3afac
+Revises: 2f4eefa1050c
+Create Date: 2015-10-28 10:25:51.199436
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '38641fb3afac'
+down_revision = '2f4eefa1050c'
+
+
+def upgrade():
+    op.add_column('forms', sa.Column('is_participation', sa.Boolean(), nullable=True), schema='event_registration')
```

Comment:
`nullable=False, server_default='false'` and then

    op.alter_column('forms', 'is_participation', server_default=None, schema='event_registration')